molecule/default/requirements.yml

```diff
@@ -1,2 +1,7 @@
 ---
-- role: ome.basedeps
+- name: basedeps
+  src: https://github.com/jburel/ansible-role-basedeps/
+  version: rocky_ubuntu
+- name: cli_utils
```

Comment:
Can you also revert this self-referencing commit to see if Molecule works as expected without it?

Reply:
fixed in https://github.com/ome/ansible-role-cli-utils/pull/9/commits/3866580bbda0a634c25a5dea6dae7fdc2e1d654f